### PR TITLE
QoL for Moongate

### PR DESCRIPTION
**Pull request by kaincenteno**
Moongate now open from 18:00 to 6:00 on full moon.
Removed semicolons at the end, and removed parenthesis on if statements.

**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/zones/RoMaeve/Zone.lua [hunk 1]
```diff
@@ -3,60 +3,55 @@
...
-    if (weather ~= tpz.weather.SUNSHINE and GetNPCByID(ID.npc.MOONGATE_OFFSET):getAnimation() ~= tpz.anim.CLOSE_DOOR) then
+    if IsMoonFull() and hour == 18 then
         for i = ID.npc.MOONGATE_OFFSET, ID.npc.MOONGATE_OFFSET+7 do
-            GetNPCByID(i):setAnimation(tpz.anim.CLOSE_DOOR);
+            GetNPCByID(i):openDoor(1728)
+            GetNPCByID(i):untargetable(true)
         end
-    elseif (weather == tpz.weather.SUNSHINE and IsMoonFull() == true and VanadielHour() < 3) then -- reactivate things for the remainder of the time until 3AM
-        local moonMinRemaining = math.floor(432 * (180 - VanadielHour() * 60 + VanadielMinute())/180) -- 180 minutes (ie 3AM) subtract the time that has passed since midnight
+    elseif IsMoonFull() and hour == 6 then
```
Comment (ibm2431) [inline scripts/zones/RoMaeve/Zone.lua]: Will the moon still be "full" after the final night? We wouldn't want to accidentally leave the doors untargetable. 😉

Comment (kaincenteno) [reply scripts/zones/RoMaeve/Zone.lua]: change to
    elseif IsMoonFull() then
as
    elseif hour == 6 and GetNPCByID(ID.npc.MOONGATE_OFFSET):untargetable() then
didnt work for some reason even tho it looks legit (i just didnt want to be valid on every single vanaday).

Comment (ibm2431) [reply scripts/zones/RoMaeve/Zone.lua]: We might be able to set up a timer when we _open_ the door, but it's probably not worth it - setting it to false once a real-life hour isn't expensive (and might actually be cheaper than adding an additional timer).

**Review by ibm2431 — APPROVED** [review]
Thanks, kain~

**Review by ghost — APPROVED** [review]

**Pull request merged**